doc/api/n-api.md

```diff
@@ -1737,11 +1737,10 @@ napi_status napi_get_value_string_utf8(napi_env env,
 - `[in] buf`: Buffer to write the UTF8-encoded string into. If NULL is passed
  in, the length of the string (in bytes) is returned.
 - `[in] bufsize`: Size of the destination buffer.
-- `[out] result`: Number of bytes copied into the buffer including the null.
-terminator. If the buffer size is insufficient, the string will be truncated
-including a null terminator.
+- `[out] result`: Number of bytes copied into the buffer, excluding the null
+terminator.
```

Comment:
> If the buffer size is insufficient, the string will be truncated including a null terminator. 

I feel like this information should be represented somewhere, maybe as part of the bufsize parameter?